src/Paket.Core/PaketConfigFiles/ProjectFile.fs

```diff
@@ -1054,10 +1055,9 @@ module ProjectFile =
                 match getTargetFramework project with 
                 | Some targetFramework ->
                     if isTargetMatchingRestrictions(restrictionList,SinglePlatform targetFramework) then
-                        if projectModel.GetLibReferences targetFramework |> Seq.isEmpty then
+                        if projectModel.GetLibReferenceFiles targetFramework |> Seq.isEmpty then
```

Comment:
the obsolete warning for this one doesn't suggest a function that will give you the same results